crates/extensions/c8y_mapper_ext/src/availability.rs

```diff
@@ -0,0 +1,255 @@
+use crate::actor::SyncStart;
+use crate::actor::TimeoutKind;
+use crate::service_monitor::get_health_status_from_message;
+use crate::service_monitor::HealthStatus;
+use std::collections::HashMap;
+use std::time::Duration;
+use tedge_actors::ChannelError;
+use tedge_actors::LoggingSender;
+use tedge_actors::Sender;
+use tedge_api::entity_store::EntityMetadata;
+use tedge_api::mqtt_topics::Channel;
+use tedge_api::mqtt_topics::EntityTopicId;
+use tedge_api::mqtt_topics::MqttSchema;
+use tedge_api::mqtt_topics::ServiceTopicId;
+use tedge_mqtt_ext::MqttMessage;
+use tedge_mqtt_ext::Topic;
+use tracing::debug;
+use tracing::info;
+
+pub type TopicWithoutPrefix = String;
```

Comment:
Why not just use the `EntityTopicId` of the lead service, as the health channel info in them are just redundant?